app/main.py

```diff
@@ -2,0 +1,21 @@
+from os import mkdir, replace
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Use: mv <source> <destination>")
+
+    source = parts[1]
+    destination = parts[2]
+
+    three = destination.split("/")
+    array = []
+    for i in range(0, len(three) - 1):
+        array.append(three[i])
+        if not os.path.exists("/".join(array)):
+            mkdir("/".join(array))
```

Comment:
The use of `"/".join(array)` for directory creation is not cross-platform. It's important to use `os.path.join` to ensure compatibility across different operating systems.